c/test/CMakeLists.txt

```
@@ -16,7 +16,7 @@ set_target_properties(test_libsbp PROPERTIES
 if(MSVC)
```

this would likely solve the issue on PBR(P) side
```suggestion
if(MSVC OR CROSS_COMPILING)
```

---

I'm not sure whether any other compiler would also break, so I've made a change to only add /usr/local/* on apple platforms